### PR TITLE
refactor: extracted SearchBox

### PR DESCRIPTION
**Pull request by andreapiras**
## 🖼 Context

This is the first step towards Select refactor

## 🚀 Changes

- Extracted search box

## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/components/src/select/searchBox/SearchBox.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  classNames: PropTypes.objectOf(PropTypes.string),
+  handleSearchChange: PropTypes.func.isRequired,
+  onClick: PropTypes.func.isRequired,
+  searchPlaceholder: PropTypes.string,
+  value: PropTypes.string,
+};
+
+SearchBox.defaultProps = {
+  classNames: {},
+  searchPlaceholder: 'Search...',
```
Comment (dozortsev) [inline packages/components/src/select/searchBox/SearchBox.js:46]: this should be in translations

Comment (dozortsev) [reply packages/components/src/select/searchBox/SearchBox.js:46]: see how same thing implemented in Upload component

Comment (andreapiras) [reply packages/components/src/select/searchBox/SearchBox.js]: Maybe we make it required and handle all translations at Select level

Comment (dozortsev) [reply packages/components/src/select/searchBox/SearchBox.js]: I'm okay with changes as long as such messages handled via our translations setup

Comment (dozortsev) [reply packages/components/src/select/searchBox/SearchBox.js]: in Select component, will `searchPlaceholder` prop be also required

Comment (andreapiras) [reply packages/components/src/select/searchBox/SearchBox.js]: It has a default searchPlaceholder: 'Search...',

Comment (andreapiras) [reply packages/components/src/select/searchBox/SearchBox.js]: The tests fails only on CI https://app.circleci.com/pipelines/github/transferwise/neptune-web/9853/workflows/a5f609f6-c461-4c34-a03f-d32341a531c3/jobs/23867

Comment (andreapiras) [reply packages/components/src/select/searchBox/SearchBox.js]: I m going to give it a default ''

**Pull request closed without merging**